.github/workflows/Operator-CI.yaml

```diff
@@ -63,20 +63,23 @@ jobs:
           kubectl create -f kustomize/overlays/minikube/keycloak/keycloak.yaml -n $NAMESPACE
           kubectl wait --for=condition=available --timeout=120s deployment/keycloak -n $NAMESPACE
           # Deploy operator
-          kubectl apply -f shard-operator/target/kubernetes/bridgeingresses.com.redhat.service.bridge-v1.yml
           kubectl apply -f shard-operator/target/kubernetes/bridgeexecutors.com.redhat.service.bridge-v1.yml
           kubectl apply -f shard-operator/target/kubernetes/minikube.yml
           # Point SSO URL to the Keycloak Service port, use HTTP to avoid complications with certificate
           kubectl set env deployment/shard-operator EVENT_BRIDGE_SSO_URL=http://keycloak:8180/auth/realms/event-bridge-fm -n $NAMESPACE
           kubectl set env deployment/shard-operator EVENT_BRIDGE_EXECUTOR_IMAGE=quay.io/5733d9e2be6485d52ffa08870cabdee0/empty-it-image:1.0 -n $NAMESPACE
-          kubectl set env deployment/shard-operator EVENT_BRIDGE_INGRESS_IMAGE=quay.io/5733d9e2be6485d52ffa08870cabdee0/empty-it-image:1.0 -n $NAMESPACE
           kubectl wait --for=condition=available --timeout=120s deployment/shard-operator -n $NAMESPACE
           # Add Prometheus ServiceMonitor CRD to avoid BridgeIngress Condition Ready: False Type: PrometheusUnavailable
           kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/v0.9.0/manifests/setup/prometheus-operator-0servicemonitorCustomResourceDefinition.yaml
       - name: Run integration tests
+        env:
+          IT_SHARD_KAFKA_BOOTSTRAP_SERVERS: ${{ secrets.IT_SHARD_KAFKA_BOOTSTRAP_SERVERS }}
+          IT_SHARD_KAFKA_USER: ${{ secrets.IT_SHARD_KAFKA_USER }}
+          IT_SHARD_KAFKA_PASSWORD: ${{ secrets.IT_SHARD_KAFKA_PASSWORD }}
+          IT_SHARD_KAFKA_TOPIC_NAME: ${{ secrets.IT_SHARD_KAFKA_TOPIC_NAME }}
         uses: ./.github/actions/maven
         with:
-          maven-command: clean verify -Pcucumber -Dparallel
+          maven-command: clean verify -Pcucumber -Dparallel -Dit.shard.kafka.bootstrap.servers=$IT_SHARD_KAFKA_BOOTSTRAP_SERVERS -Dit.shard.kafka.user=$IT_SHARD_KAFKA_USER -Dit.shard.kafka.password=$IT_SHARD_KAFKA_PASSWORD -Dit.shard.kafka.topic.name=$IT_SHARD_KAFKA_TOPIC_NAME -Dit.shard.kafka.protocol=SASL_SSL -Dit.shard.kafka.sasl.mechanism=PLAIN
```

Comment:
Is this because now the Kafka Broker directly connect to a cluster while before we were not doing this connection?